### ISSUE TITLE
Delete.update() returns 0 with instead of trigger

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sagarswathi/h2database#350]
```
The method Delete.update() always returns 0 if an instead of trigger is used.
A patch is attached.
```

Original issue reported on code.google.com by `d...@gmx.com` on 12 Oct 2011 at 8:02

**Comment by GoogleCodeExporter** [comment]
```
Hi,

Thanks a lot for the patch! This will be fixed in the next release. I have also 
added a test case.

Regards,
Thomas
```

Original comment by `thomas.t...@gmail.com` on 26 Oct 2011 at 5:56
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in version 1.3.161
```

Original comment by `thomas.t...@gmail.com` on 28 Oct 2011 at 3:56
- Changed state: **Fixed**
